test/parallel/test-http-response-close-event-race.js

```diff
@@ -0,0 +1,88 @@
+var common = require('../common');
+var assert = require('assert');
+var http = require('http');
+
+var clientRequest = null;
+var serverResponseFinishedOrClosed = 0;
+var testTickCount = 3;
+
+var server = http.createServer(function (req, res) {
+  console.log('server: request');
```

Comment:
Please keep them for now :) I've a bit delayed this, but going to review it next week!